crates/runtime/src/builder.rs

```diff
@@ -232,3 +244,26 @@ impl Default for RuntimeBuilder {
         Self::new()
     }
 }
+
+fn parse_memory_limit(memory_limit: Option<String>) -> Option<u64> {
+    let memory_limit = memory_limit?;
+    let original_memory_limit = memory_limit.clone();
+    let memory_limit = memory_limit.to_uppercase();
+    let memory_limit = if let Some(limit) = memory_limit.strip_suffix("GB") {
```

Comment:
there isn't a library for this we've been using?

Reply:
Yes, we should use the library.